classes/ClaytonTestClass3.cls

```diff
@@ -0,0 +1,16 @@
+public class ClaytonTestClass3 {
+    public void AWrongMethodN_a_me_incredible_wow(){
+        Boolean WrongFormat_VARIABLE = false;
+    }
+    
+    public void anotherMethod(){
+        for(Contact cnt : [Select Id, AccountId From Contact]){
+            for(Account acc : [Select Id From Account Where Id = :cnt.AccountId]){
```

Comment:
This SOQL statement is inside a loop and is highly likely to hit governor limits. Please move the query outside of the loop and put records in collections for processing.